<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>771</width>
    <height>345</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralWidget">
   <layout class="QHBoxLayout" name="horizontalLayout_3">
    <item>
     <widget class="QTabWidget" name="tabWidget">
      <property name="font">
       <font>
        <pointsize>11</pointsize>
       </font>
      </property>
      <property name="tabShape">
       <enum>QTabWidget::Rounded</enum>
      </property>
      <property name="currentIndex">
       <number>0</number>
      </property>
      <widget class="QWidget" name="strategy_tab">
       <attribute name="title">
        <string>策略</string>
       </attribute>
       <layout class="QHBoxLayout" name="horizontalLayout_2" stretch="1,1,1">
        <item>
         <layout class="QVBoxLayout" name="stock_inf_layout" stretch="1,1,1,1,1,1,1">
          <item>
           <layout class="QHBoxLayout" name="code_layout" stretch="1,2">
            <item>
             <widget class="QLabel" name="code_sig">
              <property name="text">
               <string>证券代码：</string>
              </property>
              <property name="textFormat">
               <enum>Qt::AutoText</enum>
              </property>
              <property name="wordWrap">
               <bool>false</bool>
              </property>
             </widget>
            </item>
            <item>
             <widget class="QComboBox" name="code">
              <property name="editable">
               <bool>true</bool>
              </property>
              <property name="currentIndex">
               <number>-1</number>
              </property>
              <property name="maxVisibleItems">
               <number>11</number>
              </property>
              <property name="duplicatesEnabled">
               <bool>false</bool>
              </property>
              <property name="modelColumn">
               <number>0</number>
              </property>
             </widget>
            </item>
           </layout>
          </item>
          <item>
           <layout class="QHBoxLayout" name="name_layout" stretch="1,2">
            <item>
             <widget class="QLabel" name="name_sig">
              <property name="font">
               <font>
                <pointsize>11</pointsize>
               </font>
              </property>
              <property name="text">
               <string>证券名称：</string>
              </property>
             </widget>
            </item>
            <item>
             <widget class="QLabel" name="name">
              <property name="text">
               <string>TextLabel</string>
              </property>
             </widget>
            </item>
           </layout>
          </item>
          <item>
           <layout class="QHBoxLayout" name="current_layout" stretch="1,2">
            <item>
             <widget class="QLabel" name="current_sig">
              <property name="font">
               <font>
                <pointsize>11</pointsize>
               </font>
              </property>
              <property name="text">
               <string>现    价：</string>
              </property>
             </widget>
            </item>
            <item>
             <widget class="QLabel" name="current">
              <property name="text">
               <string>TextLabel</string>
              </property>
             </widget>
            </item>
           </layout>
          </item>
          <item>
           <layout class="QHBoxLayout" name="holding_layout" stretch="1,2">
            <item>
             <widget class="QLabel" name="holding_sig">
              <property name="font">
               <font>
                <pointsize>11</pointsize>
               </font>
              </property>
              <property name="text">
               <string>持 有 量：</string>
              </property>
             </widget>
            </item>
            <item>
             <widget class="QLabel" name="holding">
              <property name="text">
               <string>TextLabel</string>
              </property>
             </widget>
            </item>
           </layout>
          </item>
          <item>
           <layout class="QHBoxLayout" name="cost_layout" stretch="1,2">
            <item>
             <widget class="QLabel" name="cost_sig">
              <property name="font">
               <font>
                <pointsize>11</pointsize>
               </font>
              </property>
              <property name="text">
               <string>成 本 价：</string>
              </property>
             </widget>
            </item>
            <item>
             <widget class="QLabel" name="cost">
              <property name="text">
               <string>TextLabel</string>
              </property>
             </widget>
            </item>
           </layout>
          </item>
          <item>
           <layout class="QHBoxLayout" name="profit_layout" stretch="1,2">
            <item>
             <widget class="QLabel" name="profit_sig">
              <property name="font">
               <font>
                <pointsize>11</pointsize>
               </font>
              </property>
              <property name="text">
               <string>浮动盈亏：</string>
              </property>
             </widget>
            </item>
            <item>
             <widget class="QLabel" name="profit">
              <property name="text">
               <string>TextLabel</string>
              </property>
             </widget>
            </item>
           </layout>
          </item>
          <item>
           <widget class="QPushButton" name="pushButton">
            <property name="text">
             <string>开始程序</string>
            </property>
           </widget>
          </item>
         </layout>
        </item>
        <item>
         <layout class="QVBoxLayout" name="verticalLayout">
          <item>
           <widget class="QLabel" name="selected_strategy_sig">
            <property name="font">
             <font>
              <pointsize>11</pointsize>
             </font>
            </property>
            <property name="text">
             <string>已选用策略：</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QListWidget" name="selected_strategy"/>
          </item>
          <item>
           <widget class="QLabel" name="available_strategy_sig">
            <property name="font">
             <font>
              <pointsize>11</pointsize>
             </font>
            </property>
            <property name="text">
             <string>可选用策略：</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QListWidget" name="avaliable_strategy"/>
          </item>
         </layout>
        </item>
        <item>
         <layout class="QVBoxLayout" name="description_layout">
          <item>
           <widget class="QLabel" name="description_sig">
            <property name="font">
             <font>
              <pointsize>11</pointsize>
             </font>
            </property>
            <property name="text">
             <string>策略说明：</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QTextBrowser" name="description"/>
          </item>
          <item>
           <layout class="QVBoxLayout" name="button_layout">
            <item>
             <layout class="QHBoxLayout" name="move_button_layout">
              <item>
               <widget class="QPushButton" name="up_button">
                <property name="enabled">
                 <bool>true</bool>
                </property>
                <property name="text">
                 <string>上移</string>
                </property>
               </widget>
              </item>
              <item>
               <widget class="QPushButton" name="down_button">
                <property name="text">
                 <string>下移</string>
                </property>
               </widget>
              </item>
             </layout>
            </item>
            <item>
             <layout class="QHBoxLayout" name="select_button_layout">
              <item>
               <widget class="QPushButton" name="select_button">
                <property name="text">
                 <string>选用</string>
                </property>
               </widget>
              </item>
              <item>
               <widget class="QPushButton" name="delete_button">
                <property name="text">
                 <string>删除</string>
                </property>
               </widget>
              </item>
             </layout>
            </item>
           </layout>
          </item>
         </layout>
        </item>
       </layout>
      </widget>
      <widget class="QWidget" name="diary_tab">
       <attribute name="title">
        <string>日志</string>
       </attribute>
      </widget>
      <widget class="QWidget" name="setting_tab">
       <attribute name="title">
        <string>设置</string>
       </attribute>
      </widget>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QMenuBar" name="menuBar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>771</width>
     <height>23</height>
    </rect>
   </property>
   <widget class="QMenu" name="menuStockMaster">
    <property name="title">
     <string>StockMaster</string>
    </property>
   </widget>
   <addaction name="menuStockMaster"/>
  </widget>
  <widget class="QToolBar" name="mainToolBar">
   <attribute name="toolBarArea">
    <enum>TopToolBarArea</enum>
   </attribute>
   <attribute name="toolBarBreak">
    <bool>false</bool>
   </attribute>
  </widget>
  <widget class="QStatusBar" name="statusBar"/>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources/>
 <connections/>
</ui>
